.github/workflows/integration.yaml

```diff
@@ -27,6 +27,17 @@ jobs:
           - 9091:9091
           - 9092:9092
 
+      kafka_confluent:
+        image: confluentinc/confluent-local:latest
+        hostname: broker
+        container_name: broker
```

Comment:
tests are failing due to this; needed?

Reply:
Yes. I tried to remove it but failed if we set a customized port(9192, the default is 9092) to expose the service on the workflow

Reply:
Please fix, otherwise can't approve

Reply:
@embano1 I have removed the `hostname` and `container_name` from here now.